packages/schemas/tables/daily_token_usage.sql

```diff
@@ -1,11 +1,18 @@
 create table daily_token_usage (
   id varchar(21) not null,
-  tenant_id varchar(21) not null
-    references tenants (id) on update cascade on delete cascade,
+  tenant_id varchar(21) not null,
   usage bigint not null default(0),
+  user_token_usage bigint not null default(0),
+  m2m_token_usage bigint not null default(0),
   date timestamptz not null,
   primary key (id)
 );
 
 create unique index daily_token_usage__date
   on daily_token_usage (tenant_id, date);
+
+create index daily_token_usage__user_token_usage
+  on daily_token_usage (tenant_id, date, user_token_usage);
+
+create index daily_token_usage__m2m_token_usage
+  on daily_token_usage (tenant_id, date, m2m_token_usage);
```

Comment:
Do we need to include the user_token_usage and m2m_token_usage columns in an index?

Reply:
No need to add more index, current implementation is good enough.